plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/DraggableAnnotationController.java

```diff
@@ -116,25 +121,19 @@ void onMoveEnd() {
     stopDragging(draggedAnnotation);
   }
 
-  void startDragging(@Nullable T annotation) {
-    if (annotation != null && annotation.isDraggable()) {
-      if (!annotationManager.getDragListeners().isEmpty()) {
```

Comment:
This prevents from creating unnecessary iterators, it'd be great to keep it in place.